user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -70,6 +70,7 @@ Others
 - Added new Form helper function :php:func:`validation_errors()`, :php:func:`validation_list_errors()` and :php:func:`validation_show_error()` to display Validation Errors.
 - Now you can autoload helpers by **app/Config/Autoload.php**.
 - ``BaseConnection::escape()`` now excludes the ``RawSql`` data type. This allows passing SQL strings into data.
+- SQLSRV now automatically drops ``DEFAULT`` constraint when using ``Forge::dropColumn()``.
```

Comment:
For example, this commit is not good. Because the document change has nothing to do with the code change.

We added the explanation for commits and commit messages:
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#committing